airflow/example_dags/example_http_operator.py

```diff
@@ -92,7 +92,7 @@
     http_conn_id='http_default',
     endpoint='',
     request_params={},
-    response_check=lambda response: True if "Google" in response.content else False,
```

Comment:
response.content works as well. Does it error for you @feluelle ?

Reply:
Yes. `response.content` is a byte string and not a decoded string like `response.text`.
So either `b"Google" in response.content` would work or `"Google" in response.text` . See these Jira tickets: https://issues.apache.org/jira/browse/AIRFLOW-3519 and https://issues.apache.org/jira/browse/AIRFLOW-450

Reply:
Makes sense. Thanks @feluelle

Reply:
..and I thought `"Google" in response.text` makes more sense (to me) and you do not need to do the byte conversion by yourself. :)